spark-iotdb-connector/Readme.md

```diff
@@ -0,0 +1,68 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
```

Comment:
Also put a copy of this readme under the directory `docs/Documentation/UserGuide/`.
You can name it as `9-Tools-spark-iotdb.md` and rename `9-Tools-spark.md` to `9-Tools-spark-tsfile.md`.
Chinese version of the documentation under `docs/Documentation-CHN/UserGuide/` can be left as a future work. Or you can finish it in this pr too.